src/main/java/org/apache/sysml/runtime/instructions/gpu/context/CSRPointer.java

```diff
@@ -54,7 +54,7 @@
 
 	private static final Log LOG = LogFactory.getLog(CSRPointer.class.getName());
 
-	private static final double ULTRA_SPARSITY_TURN_POINT = 0.0004;
+	private static final double ULTRA_SPARSITY_TURN_POINT = 0.00004;
```

Comment:
as a nice to have, I would prefer to simply reference MatrixBlock.ULTRA_SPARSITY_TURN_POINT for consistency.